### PR TITLE
IA-1870 Give all permissions to user created with api setupaccount

### PR DESCRIPTION
**Pull request by SimonDus**
User created with the API setupaccount are now given all permissions. 

Related JIRA tickets : IA-1870

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [x] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

User is superuser at creation.

## How to test

Add a user with the endpoint /api/setupaccount. It must be superuser.

### CODE REVIEW

File: iaso/api/setup_account.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def create(self, validated_data):
             password=validated_data["password"],
             first_name=validated_data.get("user_first_name", ""),
             last_name=validated_data.get("user_last_name", ""),
+            is_superuser=True,
```
Comment (madewulf) [inline iaso/api/setup_account.py]: This is unfortunately not the solution: this gives access to all the accounts through the admin and should NOT be given to an account admin.

Comment (madewulf) [inline iaso/api/setup_account.py]: This is unfortunately not the solution: this gives access to all the accounts through the admin and should NOT be given to an account admin.

File: iaso/api/setup_account.py [hunk 2]
```diff
@@ -12,6 +12,56 @@
 logger = logging.getLogger(__name__)
 
 
+DEFAULT_PERMISSIONS_FOR_NEW_SETUP_ACCOUNT_USER = [
```
Comment (beygorghor) [inline iaso/api/setup_account.py]: From @bmonjoie we should use constants here and i totally agree.
Also, this is not a superuser, we have to maintain this while adding new permissions.

**Review by beygorghor — CHANGES_REQUESTED** [review]

File: iaso/api/setup_account.py (hunk 2 above)
Comment (SimonDus) [reply iaso/api/setup_account.py]: I reworked it so the list comes from the model file.

Comment (bmonjoie) [reply iaso/api/setup_account.py]: @SimonDus What @beygorghor is saying is not going to be possible until my PR is merged #718

**Review by madewulf — APPROVED** [review]

**Review by hakifran — APPROVED** [review]

**Review by beygorghor — APPROVED** [review]

**Pull request merged**